### PR TITLE
docs: Update references of specific versions to use Latest Supported.

### LINKED ISSUE #160: Eliminate specific version numbers from examples and docs to extent possible
> In general, the version specific comments throughout these docs will have a short shelf life.

https://github.com/cloudevents/sdk-javascript/pull/141/files#r424509323

This is generally true. We should, as much as possible, find a way to eliminate these from the docs and examples except where it can't be avoided - e.g. providing a version number to an emitter to ensure that an event is transported via a specific spec version.

### PR DESCRIPTION
**Pull request by lholmquist**
fixes #160

Signed-off-by: Lucas Holmquist <lholmqui@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ binary or structured format, and determine what version of the CloudEvents
 specification you want to send the event as.
 
 By default, the `HTTPEmitter` will emit events over HTTP POST using the
-1.0 specification, in binary mode. You can emit 0.3 events by providing
+latest supported specification, in binary mode. You can emit version specific events by providing
```
Comment (lance) [inline README.md:57]: Suggestion: "latest supported specification version"

File: README.md [hunk 2]
```diff
@@ -54,7 +54,7 @@ binary or structured format, and determine what version of the CloudEvents
...
 the specication version in the constructor to `HTTPEmitter`. To send
 structured events, add that string as a parameter to `emitter.sent()`.
```
Comment (lance) [inline README.md:59]: Typo that can maybe corrected while you are working on this: should be `emitter.send()`.

File: README.md (hunk 1 above)
Comment (lholmquist) [reply README.md:57]: yea, that makes sense

File: README.md (hunk 2 above)
Comment (lholmquist) [reply README.md:59]: :+1:

**Comment by lholmquist** [comment]
got the nits

**Pull request merged**